release notes/v0.55.0.md

```diff
@@ -0,0 +1,214 @@
+k6 `v0.55.0` is here 🎉! This release includes:
+
+- ⚠️ The deprecated StatsD output has been removed.
+- ⚠️ The experimental `k6/experimental/tracing` module has been removed.
+- 🆕 URL grouping support in the browser module.
+- 🆕 Top-level `await` support.
+- 🔐 Complete RSA support for `k6/experimental/webcrypto`.
+
+## Breaking changes
```

Comment:
@mstoykov should we also list https://github.com/grafana/k6/pull/4009 in breaking changes?

Reply:
I guess so ... I am pretty sure nobody is using this and in practice even if they did - they very likely than gave it to another API that supports all cases